core/src/main/java/tech/pegasys/web3signer/core/Eth2Runner.java

```diff
@@ -90,6 +91,7 @@ public class Eth2Runner extends Runner {
   public static final String KEYSTORES_PATH = "/eth/v1/keystores";
   public static final String PUBLIC_KEYS_PATH = "/api/v1/eth2/publicKeys";
   public static final String SIGN_PATH = "/api/v1/eth2/sign/:identifier";
+  public static final String HIGH_WATERMARK_PATH = "/api/v1/eth2/highWatermark";
```

Comment:
Should this be under this API prefix considering it should not be part of https://ethereum.github.io/remote-signing-api ?

Reply:
Think it is fine to have it under this API prefix